libbeat/scripts/Makefile

```diff
@@ -211,17 +211,28 @@ coverage-report:
 
 # Update expects the most recent version of libbeat in the GOPATH
 .PHONY: update
-update:
-	bash ${ES_BEATS}/libbeat/scripts/update.sh ${BEATNAME} ${BEAT_DIR}/${BEATNAME} ${ES_BEATS}/libbeat
+update: python-env
```

Comment:
LGTM. That means we can actually remove update.sh? The main change is that the variables are not validated. But because of beat-generator we can assume that all the files etc. exist.